File: Electronics_Shop_17.Services/Database/AccessoryProperty.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Services.Database
{
    public class AccessoryProperty : IHelperId
    {
        public int Id { get; set; }
        public string PropertyName { get; set; }
        public string PropertyValue { get; set; }

        public int AccessoryId { get; set; }
    }
}

File: angular17_web/src/app/models/accessory-property/accessory.property.ts
export class AccessoryProperty {
  id!: number;
  propertyName!: string;
  propertyValue!: string;

  constructor(data?: any) {
    if (!data) return;

    this.id = data.id;
    this.propertyName = data.propertyName;
    this.propertyValue = data.propertyValue;
  }
}
